sympy/stats/tests/test_continuous_rv.py

```diff
@@ -33,8 +33,8 @@
 
 
 def test_single_normal():
-    mu = Symbol('mu', real=True)
-    sigma = Symbol('sigma', positive=True)
+    mu = Symbol('mu', extended_real=True)
+    sigma = Symbol('sigma', extended_positive=True)
```

Comment:
I don't think it makes sense to change these. Perhaps x below could be declared as real. I don't think it makes sense to have quantiles in a non-(extended)-real parameter.